db/seeds.rb

```diff
@@ -64,7 +60,7 @@
                         gpu: rand(50)+1,
                         ram: 4+rand(17),
                         size: 1+ rand(4),
-                        status_id: 1 + rand(5) })
+                        aasm_state: ["Released to arcade", "Not released", "Game under review", "Game not uploaded, it is not compatible with the system", "Rejected"].sample })
```

Comment:
Can you put `["Released to arcade", "Not released", "Game under review", "Game not uploaded, it is not compatible with the system", "Rejected"]` in a constant at the top of the file and refer to it instead of repeating here and below? 😎